ffi/capi/dart/package/pubspec.yaml

```diff
@@ -4,10 +4,10 @@ description: Dart bindings for ICU4X.
 repository: https://github.com/unicode-org/icu4x/tree/main/ffi/capi/dart/package
 
 environment:
-  sdk: '>=2.18.0'
+  sdk: '>=2.18.0 <4.0.0'
```

Comment:
does the linter check that this is compatible with each version? I remember seeing some errors about having to use 2.18 instead of 2.17, even though my sdk was 3.1.5.

Reply:
Yes, the linter will complain when using features from a higher version than the pubspec constraint, independent of your installed SDK